do-dont/doc/do-dont.md

```diff
@@ -104,12 +104,17 @@ export const dont_with_component = () => html`<dockit-dont>
 </dockit-do>
 ```
 
+### Do with custom icon
+
+SVGs should define `currentColor` on the needed parts, see example below.
+
 ```js preview-story
-const customIcon = svg`<svg
+const customIcon = `<svg
```

Comment:
isn't it better to use `html` literal for code highlighting?
it doesn't force users to use Lit in real life in such a case, but just makes highlighting possible

Reply:
AFAICT code highlighting doesn't work in MD docs even if we use the `html` literal, e.g.
<img width="411" alt="image" src="https://user-images.githubusercontent.com/7051248/165716921-bded2c1f-8d4b-4c42-807e-0c74bed6804e.png">

Reply:
good point, then indeed not much reasons to use `html` literal